src/System.Security.Cryptography.Algorithms/tests/HashAlgorithmTest.cs

```diff
@@ -40,17 +63,29 @@ protected void Verify(byte[] input, string output)
             {
                 Assert.True(hash.HashSize > 0);
                 actual = hash.ComputeHash(input, 0, input.Length);
-            }
 
-            Assert.Equal(expected, actual);
+                Assert.Equal(expected, actual);
+
+#if netstandard17
+                actual = hash.Hash;
+                Assert.Equal(expected, actual);
+#endif
+            }
         }
 
         protected void VerifyRepeating(string input, int repeatCount, string output)
         {
             using (Stream stream = new DataRepeatingStream(input, repeatCount))
             {
-                Verify(stream, output);
+                VerifyComputeHashStream(stream, output);
+            }
+
+#if netstandard17
+            using (Stream stream = new DataRepeatingStream(input, repeatCount))
+            {
+                VerifyICryptoTransformStream(stream, output);
             }
+#endif
```

Comment:
There are differences between netfx and corefx here. See the ActiveIssue and todo in the tests